tests/test_wave_01.py

```diff
@@ -141,12 +141,11 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) is 1
     assert len(updated_data["watched"]) is 2
-    
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert updated_data["watched"][1]["title"] is MOVIE_TITLE_1
+    assert updated_data["watched"][1]["genre"] is GENRE_1
+    assert updated_data["watched"][1]["rating"] is RATING_1  
```

Comment:
In this case I suggest `assert HORROR_1 in updated_data["watched"]`.  The requirements don't specify in what order the the movie should be added to the "watched" list.